### ISSUE TITLE
Strange behavior when switching to true fullscreen without changing the drawable size

### ISSUE THREAD

**Issue by tesselode** [ggez/ggez#865]
**Describe the bug**
If you switch to true fullscreen without changing the drawable size, the game appears as a borderless window in the top-left corner of the screen.

**To Reproduce**
Run this code and press F11:
```rust
use std::error::Error;

use ggez::{
	conf::FullscreenType,
	event::{EventHandler, KeyCode, KeyMods},
	graphics, Context, ContextBuilder, GameResult,
};
use graphics::{DrawMode, Mesh};

pub struct MainState {
	fullscreen: bool,
}

impl MainState {
	pub fn new() -> Self {
		Self { fullscreen: false }
	}
}

impl EventHandler for MainState {
	fn key_down_event(
		&mut self,
		ctx: &mut Context,
		keycode: KeyCode,
		_keymods: KeyMods,
		_repeat: bool,
	) {
		match keycode {
			KeyCode::F11 => {
				if self.fullscreen {
					graphics::set_fullscreen(ctx, FullscreenType::Windowed).unwrap();
				} else {
					graphics::set_fullscreen(ctx, FullscreenType::True).unwrap();
				}
				self.fullscreen = !self.fullscreen;
			}
			_ => {}
		}
	}

	fn update(&mut self, _ctx: &mut Context) -> GameResult {
		Ok(())
	}

	fn draw(&mut self, ctx: &mut Context) -> GameResult {
		graphics::clear(ctx, graphics::BLACK);
		let (drawable_width, drawable_height) = graphics::drawable_size(ctx);
		let circle = Mesh::new_circle(
			ctx,
			DrawMode::fill(),
			[drawable_width / 2.0, drawable_height / 2.0],
			drawable_height / 2.0,
			0.1,
			graphics::WHITE,
		)?;
		graphics::draw(ctx, &circle, ([0.0, 0.0],))?;
		graphics::present(ctx)?;
		Ok(())
	}
}

fn main() -> Result<(), Box<dyn Error>> {
	let (ctx, event_loop) = ContextBuilder::new("test", "test").build()?;
	let main_state = MainState::new();
	ggez::event::run(ctx, event_loop, main_state);
}
```

**Expected behavior**
The desktop resolution should switch to whatever the drawable size is, and the game should fill the entire screen (stretched out).

**Screenshots or pasted code**
Windowed mode:
![image](https://user-images.githubusercontent.com/2637802/103609468-f6cf6480-4eeb-11eb-8b51-44332922aa63.png)

Fullscreen:
![image](https://user-images.githubusercontent.com/2637802/103609488-fe8f0900-4eeb-11eb-996d-63109a4599de.png)

**Hardware and Software:**
 - ggez version: current git devel
 - OS: Windows 10
 - Graphics card: NVidia GTX 970

**Comment by icefoxen** [comment]
Needs confirmation, but good catch, thanks.

**Comment by PSteinhaus** [comment]
fixed in #884
